### DISCUSSION TITLE
What Are the Three Stages of Building a Model in Machine Learning?

### DISCUSSION THREAD

**Discussion by luffy-Monkey123** [luffy-Monkey123/python-cool-programs#19] [Q&A] [answered]
What Are the Three Stages of Building a Model in Machine Learning?

**Comment by RUTAGENGWA-ASANTE-BRUCE** [comment, marked as answer]
The three stages of building a[ machine learning model](https://www.simplilearn.com/machine-learning-models-article) are:

1. Model Building
Choose a suitable algorithm for the model and train it according to the requirement 
2. Model Testing
Check the accuracy of the model through the test data 
3. Applying the Model
Make the required changes after testing and use the final model for real-time projects
Here, it’s important to remember that once in a while, the model needs to be checked to make sure it’s working correctly. It should be modified to make sure that it is up-to-date.
